rest/taskana-rest-spring/src/main/java/pro/taskana/common/rest/ldap/LdapClient.java

```diff
@@ -34,26 +33,22 @@
 @Component
 public class LdapClient {
 
-  static final String MISSING_CONFIGURATION_S =
-      "LdapClient was called but is not active due to missing configuration: %s ";
-
   private static final Logger LOGGER = LoggerFactory.getLogger(LdapClient.class);
-
   private static final String CN = "cn";
 
+  private final Environment env;
+  private final LdapTemplate ldapTemplate;
   private boolean active = false;
-
-  @Autowired private Environment env;
-
-  @Autowired(required = false)
-  private LdapTemplate ldapTemplate;
-
   private int minSearchForLength;
-
   private int maxNumberOfReturnedAccessIds;
-
   private String message;
 
+  @Autowired
```

Comment:
Just fyi: As of Spring 4.3, you no longer need to specify an explicit injection annotation in a single-constructor scenario. (meaning the @Autowired could be deleted here)